Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/ToolBar/EditorToolBar.cpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/ActionManager/ToolBar/EditorToolBar.h>
+#include <AzToolsFramework/ActionManager/Action/ActionManagerInterface.h>
+#include <AzToolsFramework/ActionManager/ToolBar/ToolBarManagerInterface.h>
+
+#include <QToolBar>
+
+namespace AzToolsFramework
+{
+    EditorToolBar::EditorToolBar()
+        : m_ToolBar(new QToolBar(""))
```

Comment:
uber nit: `m_ToolBar` -> `m_toolBar`

Reply:
Whoops, ended up doing some last minute search/replace and I must have made a mistake lol. Fixing!